CityOfDetroit/bloom
# fix: error when passing empty string as phone number to usToIntlPhone…

… helper

- Closes #1123 

## Description

Despite phone number not being required to create a new account, an invalid phone number error was being thrown when a user does not include a phone number. This PR fixes that error.

## How Can This Be Tested/Reviewed?

Entry Point: http://localhost:3000/create-account

Create a new account with all required fields. Do not include a phone number. Sign up should be successful. 

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
